ssl/t1_trce.c

```diff
@@ -1697,6 +1697,10 @@ static int ssl_print_handshake(BIO *bio, const SSL_CONNECTION *sc, int server,
             return 0;
         break;
 
+    case SSL3_MT_END_OF_EARLY_DATA:
```

Comment:
`BIO_indent` doesn't seem quite correct when its not then followed by printing any data. You could combine the handling with that of SSL3_MT_SERVER_DONE which seems very similar. e.g. modify that case to read:

```` c
    case SSL3_MT_END_OF_EARLY_DATA:
    case SSL3_MT_SERVER_DONE:
        if (msglen != 0)
            ssl_print_hex(bio, indent + 2, "unexpected value", msg, msglen);
        break;
````

Reply:
Please combine the cases as @mattcaswell suggested. In such case it can be accepted with CLA: trivial.

Reply:
Those two cases are combined now with https://github.com/openssl/openssl/pull/25026/commits/716b1d5f9ac7295306027c35e7127829ce7a4941.